test/Microsoft.TemplateEngine.Orchestrator.RunnableProjects.UnitTests/SchemaTests/PostActionTest.json

```diff
@@ -30,6 +30,32 @@
       "actionId": "210D431B-A78B-4D2F-B762-4ED3E3EA9025",
       "continueOnError": true
     },
+    {
+      "description": "Restore NuGet packages required by this project.",
+      "manualInstructions": [
+        {
+          "text": "Run 'dotnet restore'"
+        }
+      ],
+      "actionId": "210D431B-A78B-4D2F-B762-4ED3E3EA9025",
+      "continueOnError": true,
+      "args": {
```

Comment:
I suggest updating the documentation page accordingly: https://github.com/dotnet/templating/wiki/Post-Action-Registry#restore-nuget-packages. At the moment there is no example of how to use "args" and also no information concerning the "files" property there.